### ISSUE TITLE
/morph command does not apply morph attributes (abilities, attack, etc)

### ISSUE THREAD

**Issue by asanetargoss** [asanetargoss/metamorph#12]
playful_waffle on Curseforge: "Where if you use the /morph command, You turn into the mob, but you don't get any of its attributes  (Immunity, Burn in sun, powers etc)"

Reported on version changeling-1.1.10-1.12.2_squid-air-mp.

**Comment by hoklai1997** [comment]
I think I'm experiencing the same issue, so to add a bit it seems as though when using /morph you don't get any of the abilities, also, custom-defined attributes in the morphs.json file are ignored (including health)

**Comment by asanetargoss** [comment]
By any chance, does the configuration get applied properly when acquiring the morph in the creative morph menu, and morphing using the GUI?

**Comment by hoklai1997** [comment]
yes, it seems to, that has been the only way I was acquiring morphs
